src/components/login/LoginForm.jsx

```diff
@@ -5,18 +5,60 @@ import {ActivityIndicator, StyleSheet, Text, TextInput, TouchableOpacity, View}
 import {useDispatch} from 'react-redux';
 import {initSendbird} from '../../redux/slices/sendbird';
 
-import {APP_ID, COLORS, TOKEN, USER_ID} from '../../constants';
+import {COLORS} from '../../constants';
 
 export default function LoginForm() {
   const dispatch = useDispatch();
   const navigation = useNavigation();
   const [state, setState] = useState({
-    appId: APP_ID,
-    userId: USER_ID,
-    token: TOKEN,
+    appId: '',
+    userId: '',
+    token: '',
+    channelUrl: '',
     isLoading: false,
+    hasError: false,
+    errorMessage: '',
   });
 
+  useEffect(() => {
+    AsyncStorage.getItem('loginInformation').then(data => {
+      data = JSON.parse(data);
+      setState({
+        appId: data?.appId || '',
+        userId: data?.userId || '',
+        token: data?.token || '',
+        channelUrl: data?.channelUrl || '',
+        isSignedIn: data?.isSignedIn || false,
+      });
+    });
+  }, []);
+
+  useEffect(() => {
+    if (state.isSignedIn) {
+      setState({
+        ...state,
+        isLoading: true,
+      });
+      dispatch(initSendbird(state))
+        .unwrap()
+        .then(() => {
+          setState({
+            ...state,
+            isLoading: false,
+          });
+          navigation.navigate('Home');
```

Comment:
The screen will be changed by the 'isAuthenticated' ternary in the navigator (App.jsx), so we don't need to navigate manually.